changelog.md

```diff
@@ -1,4 +1,90 @@
 # v1.4.0 - yyyy-mm-dd
+# 1.2 - xxxx-xx-xx
```

Comment:
Why is the change log showing these changes?

Reply:
it's a consequence of merging https://github.com/nim-lang/Nim/pull/14098 without https://github.com/nim-lang/Nim/pull/14025
I've fixed it, but this could/will happen to other old PR's (pre-1.2) that rebase against devel. We should really re-open https://github.com/nim-lang/Nim/pull/14025 (and address whatever comments need to be addressed) or at least for time being `git move changelog.md changelogs/changelog_1_4.md` (but my PR was more careful about that and did other fixes)